automated install/basic-install.sh

```diff
@@ -1940,14 +2107,14 @@ Your Admin Webpage login password is ${pwstring}"
     fi
 
     # Final completion message to user
-    whiptail --msgbox --backtitle "Make it so." --title "Installation Complete!" "Configure your devices to use the Pi-hole as their DNS server using:
-
-IPv4:	${IPV4_ADDRESS%/*}
-IPv6:	${IPV6_ADDRESS:-"Not Configured"}
-
-If you have not done so already, the above IP should be set to static.
-
-${additional}" "${r}" "${c}"
+    dialog --no-shadow --keep-tite \
+        --title "Installation Complete!" \
+        --msgbox "Configure your devices to use the Pi-hole as their DNS server using:\
+\\n\\nIPv4:	${IPV4_ADDRESS%/*}\
+\\nIPv6:	${IPV6_ADDRESS:-"Not Configured"}\
+\\nIf you have not done so already, the above IP should be set to static.\
+\\n${additional}"
```

Comment:
`$additional` is set in line **2104**.

It should be changed to:
```bash
        additional="View the web interface at http://pi.hole/admin or http://${IPV4_ADDRESS%/*}/admin\
\\n\\nYour Admin Webpage login password is ${pwstring}"
```
to avoid problems with line breaks.